sql/src/test/java/org/opensearch/sql/sql/parser/AstExpressionBuilderTest.java

```diff
@@ -328,6 +329,14 @@ public void canBuildQualifiedNameHighlightFunction() {
     );
   }
 
+  @Test
+  public void canBuildPositionFunction() {
+    assertEquals(
+            position(AstDSL.qualifiedName("fieldA"), AstDSL.qualifiedName("fieldB")),
+            buildExprAst("position(fieldA IN fieldB)")
+    );
```

Comment:
we should test this with string literals too.

Reply:
Addressed in 9a6b7b5d8962cff8af410638da5151e9ee6a36b9